setup.cfg

```diff
@@ -20,9 +20,37 @@
 # granted to it by virtue of its status as an Intergovernmental Organization
 # or submit itself to any jurisdiction.
 
-#
+# Packaging
+
+[metadata]
+name = inspire-mitmproxy
+author = CERN
+author_email = admin@inspirehep.net
+license = GPLv3
+platforms = any
+classifiers =
+    Intended Audience :: Developers
+    Environment :: Console
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Topic :: Internet :: WWW/HTTP
+    Topic :: Software Development :: Libraries :: Application Frameworks
+    Topic :: Software Development :: Libraries :: Python Modules
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+
+[options]
+zip_safe = False
+include_package_data = True
+packages = find:
+scripts = scripts/vcr_convert.py
+
+[options.package_data]
+* = AUTHORS, CHANGELOG, entrypoint.py
```

Comment:
We need also the MANIFEST.in, for example: https://github.com/david-caro/python-foreman/blob/master/MANIFEST.in